packages/edit-site/src/components/editor/popover-wrapper.js

```diff
@@ -0,0 +1,57 @@
+/**
```

Comment:
This file is duplicated between edit-post and edit-site. Separately, I'd like to factorize that and transform it to a React hook in `@wordpress/compose` if possible.

Reply:
That sounds a good idea 👍

Reply:
Actually it's used in `edit-widgets` as well, so it's triplicate.